Xamarin.Forms.Core/Routing.cs

```diff
@@ -68,17 +78,21 @@ public static string GetRoute(Element obj)
 
 		public static void RegisterRoute(string route, RouteFactory factory)
 		{
-			if (!ValidateRoute(route))
-				throw new ArgumentException("Route must contain only lowercase letters");
+			ValidateRoute(route);
 
+#if NETSTANDARD1_0
+			route = route.ToLowerInvariant();
```

Comment:
Why to lower invariant on netstandard 1?

Reply:
because the non-ns10 has a StringComparer, see ^^^^

Reply:
I don't think it should anymore though. I'm pretty sure those just got added when I was proposing casing shouldn't matter but then EZ opened my eyes to why it has to matter 👀